src/ReactiveDomain.Users.Tests/ClientStoreTests.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ReactiveDomain.Users.Tests
+{
+    internal class ClientStoreTests
```

Comment:
Is this to-be-implemented or can it be deleted?

Reply:
it's on the checklist to be done :-)